### PR TITLE
rating component done

### PR DESCRIPTION
**Pull request by govardhanreddyc79**
Rating component

### CODE REVIEW

**Review by pariyar07 — APPROVED** [review]
Clean code looks good. Go ahead and merge.

File: Components/Rating/rating.html [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+                style="width: 100%; height: 393px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (Aditi0505) [inline Components/Rating/rating.html:69]: Please avoid inline styles.

**Review by Aditi0505 — APPROVED** [review]
Just left one suggestion.
Rest looks good. 
- Good class names
-  Readable code 
-  Good use of semantic tags

File: Components/Rating/rating.html (hunk 1 above)
Comment (govardhanreddyc79) [reply Components/Rating/rating.html:69]: sure

**Pull request merged**